src/main/scala/com/databricks/labs/overwatch/pipeline/Gold.scala

```diff
@@ -363,6 +381,71 @@ class Gold(_workspace: Workspace, _database: Database, _config: Config)
     }
   }
 
+  private def verboseAudit(): Unit = {
+    verboseAuditModule.execute(appendVerboseAuditProccess)
+    GoldTargets.verboseAuditViewTarget.publish(verboseAuditTargetViewColumnMapping)
+  }
+  private def publishVerbose():Unit = {
+    try{
+    if((spark.catalog.tableExists(s"${config.consumerDatabaseName}.notebook"))
+      & (spark.catalog.tableExists(s"${config.consumerDatabaseName}.clusterstatefact"))) {
+      val notebookLookupTSDF = spark.sql(s"SELECT * FROM ${config.consumerDatabaseName}.notebook")
+        .select("organization_id", "notebook_id", "notebook_path", "notebook_name", "unixTimeMS")
+        .withColumnRenamed("notebook_id", "notebookId")
+        .distinct
+        .toTSDF("unixTimeMS", "organization_id", "notebookId")
+
+      val clsfLookupTSDF = spark.sql(s"SELECT * FROM ${config.consumerDatabaseName}.clusterstatefact")
+        .select(
+          "organization_id", "state_start_date", "unixTimeMS_state_start", "cluster_id", "cluster_name",
+          "custom_tags", "node_type_id", "current_num_workers", "uptime_in_state_H", "total_cost")
+        .distinct
+        .withColumnRenamed("state_start_date", "date")
+        .withColumnRenamed("unixTimeMS_state_start", "unixTimeMS")
+        .withColumnRenamed("cluster_id", "clusterId")
+        .withColumnRenamed("current_num_workers", "node_count")
+        .withColumn("totalCostPMS", col("uptime_in_state_H") / col("total_cost") / lit(60) / lit(60) / lit(1000))
+        .toTSDF("unixTimeMS", "organization_id", "date", "clusterId")
+
+      val ColNames = Array(
```

Comment:
what is shardName and why is it here? It seems to always be null and shouldn't it basically be the same thing as workspace_name (semantically)? Suggest remove this field and only use workspace_name